packages/component-library/draft/Kaizen/HeroCard/README.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: HeroCard
+navTitle: HeroCard
+summaryParagraph: HeroCards are a large structural component to organise primary content.
```

Comment:
@ellisjas we use US spelling in our product and therefore the design system so "organise" will need to be "organize". 

To further distinguish Hero Cards from Cards, it's worth noting in this summary that a hero card is used to create visual interest and draw attention.